docs/source/gbd2019_models/risk_risk_correlation/birthweight_wasting/index.rst

```diff
@@ -114,111 +92,88 @@ Links to documentation for relevant risk exposure pages include:
-
-1. Assign the simulant a birthweight exposure value specific to their age group
-
-2. Assess the percentile of a simulant's birthweight exposure value within the overall age, sex, location, and year-specific birthweight exposure distribution (also known as the simulant's birthweight propensity score value *at birth*)
-
-.. todo::
-
-   Determine if the birthweight propensity score value should be assessed according to the birthweigtht distribution *at birth* or *among the current age group*. This will depend on how the correlation coefficients from the MAl-ED study were calculated by Chris Troeger.
-
-   For now, assume that the correlation coefficient is calculated among *living* children (because otherwise there would be no observed wasting values), which would suggest the use of the birthweight distribution of the current age group for calculation of the birthweight propensity score, as described above.
-
-3. Assign the simulant a wasting propensity score that is correlated to their birthweight propensity score value according to the spearman correlation coefficient of birthweight and wasting at one month of age (0.289376997). This should be done according to the methodology described in the :ref:`risk-risk correlation proposal page <2017_risk_models>`.
-
-.. todo::
+   Wasting exposures at birth and in the early and late neonatal age groups will be initialized using the wasting exposure distribution for the 6-11 month age group, as described in the :ref:`wasting exposure model document <2020_risk_exposure_wasting_state_exposure>`.
 
-   Add more detail on the methodology here/as implemented in the BEP project
+1. Assign the simulant a birthweight exposure value specific to their age group and additional attributes that affect birthweight (i.e. intervention coverage, maternal BMI risk exposure, etc.)
 
-.. note::
+2. Assess the percentile of a simulant's birthweight exposure value (after application of all birthweight-affecting variables in the baseline scenario but before the application of *new* intervention coverage in the alternative scenario) within the overall sex, location, and year-specific birthweight exposure distribution *at birth*.
 
-   The correlation coefficient of one month of age should be used for all simulants upon birth as well as those initialized into the early neonatal or late neonatal age groups because we do not have correlation coefficients specific to more detailed age ranges. 
+3. Assign the simulant a wasting propensity score that is correlated to their birthweight percentile according to the spearman correlation coefficient of birthweight and wasting at six months of age (mean: 0.3083092, standard deviation: 0.2225747, normal distribution of uncertainty). This should be done according to the methodology described in the :ref:`risk-risk correlation proposal page <2017_risk_models>`.
```

Comment:
Just a general comment: It looks like "propensity score" has a technical meaning that I think is different from the way we use the term "propensity" in our models (though it's possible they're related -- I can't really tell on a cursory reading -- see articles below), so we might want to be a bit more careful with our terminology.

https://en.wikipedia.org/wiki/Propensity_score_matching
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3144483/